.travis.yml

```diff
@@ -22,10 +22,13 @@ before_install:
     - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi;
     - phpunit --self-update
     - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
+    - npm install -g swagger-cli
```

Comment:
outch, IMO it would be better to have it in a `packages.json` and shrinkwrap it. `node_modules` could then be cached like it is for composer